search-app/docs/api.md

```diff
@@ -2020,6 +2022,28 @@ The first two values of the parameter together define the temporal bounds. See u
 
 For temporal range search, the default is inclusive on the range boundaries. This can be changed by specifying `exclude_boundary` option with `options[temporal][exclude_boundary]=true`. This option has no impact on periodic temporal searches.
 
+#### <a name="g-cycle"></a> Find granules by cycle
+
+Cycle is part of the track information of the granule. Track information is used to allow a user to search for granules whose spatial extent is based on an orbital cycle, pass, and tile mapping. Cycle must be a positive integer.
+
+User can search granules by one or more cycles. e.g.
+
+    curl -g "%CMR-ENDPOINT%/granules?cycle[]=1&cycle[]=2"
```

Comment:
We should not have any all granule searches in our documentation. Update all of the examples to include a collection.
`collection_concept_id=%CMR-EXAMPLE-COLLECTION-ID%`